packages/workflow/src/features/registration/fhir/fhir-bundle-modifier.ts

```diff
@@ -167,7 +167,7 @@ export async function invokeRegistrationValidation(
   token: string
 ) {
   try {
-    fetch(`${RESOURCE_SERVICE_URL}validate/registration`, {
+    await fetch(`${RESOURCE_SERVICE_URL}validate/registration`, {
```

Comment:
This was added so that possible errors wouldn't go uncaught

Reply:
@rikukissa I am testing this again today based on our hackathon activities and the discussion we had on Slack regarding rejecting the declaration on a country config error. If the country config service throws a 500 it is still not being caught here. I am working on a fix